roles/idrac_gather_facts/tests/asserts/secureboot_assert.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Fetching Secureboot info
+  ansible.builtin.uri: &uri_params
+    url: "https://{{ hostname }}{{ api_system
+      }}/SecureBoot/?$expand=*($levels=1)"
+    validate_certs: "{{ validate_certs }}"
+    ca_path: "{{ ca_path | default(omit) }}"
+    method: "{{ idrac_gather_facts_uri_method }}"
+    user: "{{ username }}"
+    password: "{{ password }}"
+    headers: "{{ idrac_gather_facts_uri_headers }}"
+    body_format: "{{ idrac_gather_facts_uri_body_format }}"
+    status_code: "{{ idrac_gather_facts_uri_status_code }}"
+    return_content: "{{ idrac_gather_facts_uri_return_content }}"
+  register: secureboot_details
+
+- name: Fetching Secureboot database info
+  ansible.builtin.uri:
+    url: "https://{{ hostname }}{{ api_system
+      }}/SecureBoot/SecureBootDatabases?$expand=*($levels=1)"
+    <<: *uri_params
+  register: secureboot_database_details
+
+- name: Fetching Secureboot database certificates info
+  ansible.builtin.uri:
+    url: "https://{{ hostname }}{{
+      sec_boot_db_item['Certificates']
+      ['@odata.id'] }}?$expand=*($levels=1)"
+    <<: *uri_params
+  loop: "{{ secureboot_database_details.json.Members }}"
+  loop_control:
+    loop_var: sec_boot_db_item
+  register: secure_boot_results
```

Comment:
can you rename this register var to idrac_gather_facts_

Reply:
Done